### PR TITLE
IS-1023: Set logs bloom to base tx result

### PR DESCRIPTION
**Pull request by soobokjin**
Define revision 10.
after revision change to 10, logs bloom will be added to base transaction results.

* All units, integrate tests have been passed.

### CODE REVIEW

File: iconservice/icon_constant.py [hunk 1]
```diff
@@ -115,7 +115,8 @@ class Revision(Enum):
     OPTIMIZE_DIRTY_PREP_UPDATE = 8
 
     FIX_EMAIL_VALIDATION = 9
-    LATEST = 9
+    ADD_LOGS_BLOOM_ON_BASE_TX = 10
```
Comment (leeheonseung) [inline iconservice/icon_constant.py]: I think you have to check released revision.
because if it doesn't release yet, you should set revision 9 not 10

Comment (soobokjin) [reply iconservice/icon_constant.py]: I have checked the release notes and found that the current release revision is 8. I'm going to modify the revision number to 9.

Comment (soobokjin) [reply iconservice/icon_constant.py]: Also, I have confirmed this issue to the leader.

File: iconservice/icon_service_engine.py [hunk 2]
```diff
@@ -924,8 +924,7 @@ def _invoke_genesis(self,
 
         return tx_result
 
-    @classmethod
-    def _process_base_transaction(cls,
+    def _process_base_transaction(self,
```
Comment (goldworm-icon) [inline iconservice/icon_service_engine.py]: I am curious about why _process_base_transaction() is changed to an instance method.
Is it impossible to call cls._generate_logs_bloom() or any other reason?

Comment (soobokjin) [reply iconservice/icon_service_engine.py]: As you said, there are lots of ways of calling '_generate_logs_bloom', but I just thought that making _process_base_transaction as protected instance method and calling 'self._generate_logs_bloom()' is the proper way

**Pull request merged**